dev/test/commands/generate_docs_command_test.rb

```diff
@@ -0,0 +1,189 @@
+# frozen_string_literal: true
+
+require "test_helper"
+require "tmpdir"
+
+module ProductTaxonomy
+  class GenerateDocsCommandTest < TestCase
+    setup do
+      @tmp_path = Dir.mktmpdir
+      @original_data_path = ProductTaxonomy::DATA_PATH
+      ProductTaxonomy.send(:remove_const, :DATA_PATH)
+      ProductTaxonomy.const_set(:DATA_PATH, @tmp_path)
```

Comment:
Should we make this not a const and a method instead? Just so testing is simpler?